src/components/Content/News/NewsDetails/NewsDetails.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import styled from "styled-components";
+import {Image} from "antd";
+import MockImage from "../../../../assets/images/Church.png";
+
+const Data = {
+    "news_content_id": 320,
+    "title": "Масленичное гуляние в приходе",
+    "text": "    Обряд величания молодоженов, масленичные песни и народные танцы, веселые конкурсы и, конечно, блинное изобилие с чаем из горячего самовара ожидали всех прихожан и гостей на Масленичном гулянии. Особыми гостями праздника стали воспитанники социально-педагогического центра с приютом Ленинского района г.Минска. Ребята с удовольствием приняли участие во всех конкурсах на силу, быстроту и меткость, а также тепло пообщались за чаем с учащимися воскресной школы прихода. Фоторепортаж",
+    "view_data": null,
+    "main_asset_url": "https://chashaby.s3.eu-central-1.amazonaws.com/photo_2023-03-26_21-23-44.jpg",
+    "assets_url": [
+        "https://chashaby.s3.eu-central-1.amazonaws.com/photo_2023-03-26_21-23-44.jpg",
+        "https://chashaby.s3.eu-central-1.amazonaws.com/photo_2023-03-26_21-34-54.jpg",
+    ],
+    "created_date": "2023-03-26T18:32:51",
+    "created_by_id": 1,
+    "updated_date": "2023-11-29T15:42:20",
+    "updated_by_id": 1
+}
+
+const NewsItemDetailsWrapper = styled.div`
+    flex: 25rem;
+
+    overflow-y: scroll;
+    height: 100vh;
+
+    display: grid;
+    grid-template-columns: 48% 48%;
```

Comment:
Ohhhhh I think I know how to easily animate width of news details so it's like it's flying from the right :) Just like in Sidebar new version. Remind me later if you have problems with implementation.

Reply:
I can't say that I understand the key point of the comment. Could you please say it in other words?)

Reply:
If we decide to animate the News details coming from the right side, when we click a news in News list. I might know how to animate that

Reply:
Ooh, yeah, nice idea! I will try to implement that)